doc/docs/master/getting_started/install-pachctl-completion.md

```diff
@@ -0,0 +1,55 @@
+# Install `pachctl` Autocompletion
+
+Pachyderm autocompletion allows you to automatically finish
+partially typed commands by pressing `TAB`. Autocompletion needs
+to be installed separately when `pachctl` is already
+available on your client machine.
+
+Pachyderm autocompletion is supported for both `bash` and `zsh`.
+You must have either of them preinstalled
+before installing Pachyderm autocompletion.
+
+!!! tip
+    Type `pachctl completion --help` to display help information about
+    the command.
+
+To install `pachctl` autocompletion, perform the following steps:
```

Comment:
these following instructions are for installing bash completions. do we want to have info about how to install zsh completions as well?

Reply:
added the zsh instructions